src/assets/strats/v2.0.0/MangroveOrder-Router.json

```diff
@@ -0,0 +1,348 @@
+{
+  "$schema": "../../contract.schema.json",
+  "released": false,
+  "contractName": "SmartRouter",
+  "version": "2.0.0",
```

Comment:
This is a deployment of a pre-release, not the actual v2.0.0, right?
If so, this version number (and the folder name) should reflect that pre-release version number.